opensir/models/model.py

```diff
@@ -37,6 +37,8 @@ def __init__(self):
         self.pop = None
         self.w0 = None
         self.pcov = None
+        self.fit_input = None
```

Comment:
This is a class dependent constant, not an instance dependent constant. Thus, it should be modeled as the CSV_ROW or NUM_PARAMS parameters